docs/development/local-development.md

```diff
@@ -14,7 +14,7 @@ You need the following dependencies for local development:
 - Node.js `>=14.15.4`
 - Yarn `^1.22.5`
 - C++ compiler
-- Python `^3.8`
+- Python `^3.9`
```

Comment:
Do we want to *insist* that only python 3.9 is supported?

Reply:
I thought: if I bump the Python version in our CI tests, I should also bump the development docs. That seemed the right thing to do.

If you're happy for developers to use `^3.8` I can revert that change. 😉

Reply:
@JamieMagee @viceice what are your thoughts? Should we say Python 3.9 for developers because that simplifies life for us, rather than because it's necessary?

Reply:
🤔 I'm for simplicity. So you can use older versions, but if any issue, you need to try the mentioned version.